libs/sdk-ui-kit/src/@ui/UIListbox/UIListbox.scss

```diff
@@ -0,0 +1,77 @@
+// (C) 2025 GoodData Corporation
+
+@use "../../../styles/scss/variables";
+
+.gd-ui-kit-listbox {
+    $root: &;
+
+    max-height: 300px;
+    overflow-y: auto;
+    overflow-x: hidden;
+    width: 100%;
+    min-width: 200px;
+
+    @media #{variables.$small-only} {
+        // !important to override inline style from maxWidth prop
+        /* stylelint-disable-next-line declaration-no-important */
+        max-width: 100vw !important;
+        max-height: 100vh;
+        width: 100vw;
+    }
+
+    &__items {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+        outline: none;
+
+        &:focus-visible #{$root}__item--isFocused {
```

Comment:
focus visible style should be default one. Only Input , text area and few others have our custom